t/selenium/Vote_And_Note_Submission.json5

```diff
@@ -0,0 +1,73 @@
+{
+  title: 'Vote and note submission',
+  commands: [
+    {
+      command: 'open',
+      target: '/user/editor1/edits',
+      value: '',
+    },
+    // We want to vote No on the first edit
+    {
+      command: 'click',
+      target: 'xpath=//*[@id="id-enter-vote.vote.0.vote-No"]',
+      value: '',
+    },
+    // We want to vote Yes on the second edit
+    {
+      command: 'click',
+      target: 'xpath=//*[@id="id-enter-vote.vote.1.vote-Yes"]',
+      value: '',
+    },
+    // We want to leave a valid note on the first edit
+    {
+      command: 'click',
+      target: 'xpath=//*[@id="edits"]/form/div[2]/div[2]/div[2]/a[1]',
```

Comment:
These are fine for now, but thought I'd note for the future: these types of selectors can be fragile, since if we ever change the structure of the HTML at all, or if the order of the elements changes, they will break. Where possible I stick with IDs/class names that are unlikely to change, and avoid enforcing a specific HTML structure by selecting direct descendants (so, preferring `//` instead of `/`, and specifying the least amount of descendants needed to identify the node).

Reply:
For example, assuming this is for the add-note button: a good idea might be to add IDs to each of the edit-list* divs and do something like `xpath=//div[@id="edit-123"]//a[text()="Add note"])`.

(* Why is this called `edit-list` when it contains only a single edit? 😕)

Reply:
I just used the xpath Chrome gave me for now. Better wouldn't hurt, although it seems "better IDs would be useful" is honestly true of most of our code (and tests) :)